source/CardGames.py

```diff
@@ -329,10 +329,15 @@ def updateLeaderboard(player):
 def showLeaderboard():
     with open(leaderboard_file, 'r') as file:
```

Comment:
This looks good! Really clean and makes it easy for my checkLeaderboard function to scan the txt file.

Reply:
Thanks bro! I appreciate it.